codes/data/usa/merge_policy_and_cases.py

```diff
@@ -354,11 +357,12 @@ def add_countries(x):
 			cases_data_subset.drop([key], axis=1).reset_index(drop=True)
 
 	# add in population 
-	pops1 = pd.read_csv(os.path.join(str(int_data_dir).replace('usa','adm') , 'adm1/adm1.csv'), index_col = [0,1])
+	pops1 = pd.read_csv(os.path.join(int_data_dir.replace('usa','adm') , 'adm1/adm1.csv'), index_col = [0,1])
```

Comment:
If we do go with the cutil Path objects, will need to either cast them to str at the top of the script or here.

Reply:
i cast them at the top